src/Tribe/Metabox.php

```diff
@@ -474,7 +475,13 @@ public function has_permission( $post, $data, $nonce_action ) {
 			$post = get_post( $post );
 		}
 
-		return ! empty( $data['nonce'] ) && wp_verify_nonce( $data['nonce'], $nonce_action ) && current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
+		if ( empty( $data['nonce'] ) ) {
+			return false;
+		}
+
+		$has_permission = current_user_can( 'edit_event_tickets' ) || current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
```

Comment:
So this one can be converted into:

```suggestion
		return current_user_can( 'edit_event_tickets' ) || current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
```